database/seed.js

```diff
@@ -0,0 +1,124 @@
+{"productName":"Uruguayan Fountaingrass","productId":69,"features":["Open-source systematic support","Persistent tertiary moderator","Reactive value-added product","Adaptive regional orchestration","Compatible global flexibility","Virtual dedicated orchestration","Front-line web-enabled adapter","Profit-focused mission-critical budgetary management","Innovative local conglomeration"],"techSpecs":[{"type":"Ergonomic","description":"Cloned","measurement":false},{"type":"well-modulated","description":"Inverse","measurement":false},{"type":"Cloned","description":"data-warehouse","measurement":false},{"type":"full-range","description":"methodology","measurement":false},{"type":"Cloned","description":"exuding","measurement":false},{"type":"Universal","description":"internet solution","measurement":true},{"type":"multi-state","description":"24 hour","measurement":false}]},
+{"productName":"Early Yellowrocket","productId":70,"features":["Robust bandwidth-monitored frame","Distributed radical strategy","Pre-emptive user-facing capacity","Streamlined system-worthy monitoring","Organic context-sensitive algorithm","Implemented needs-based core","Organized methodical capacity"],"techSpecs":[{"type":"Digitized","description":"Self-enabling","measurement":false},{"type":"Fully-configurable","description":"algorithm","measurement":true},{"type":"context-sensitive","description":"Reverse-engineered","measurement":true},{"type":"encoding","description":"solution-oriented","measurement":false},{"type":"real-time","description":"non-volatile","measurement":true}]},
+{"productName":"Desert Beardtongue","productId":71,"features":["Proactive bottom-line concept","Down-sized eco-centric Graphic Interface","Customer-focused contextually-based portal","Monitored fault-tolerant time-frame","Switchable zero tolerance encoding","Implemented homogeneous Graphic Interface","Visionary heuristic synergy"],"techSpecs":[{"type":"client-driven","description":"4th generation","measurement":true},{"type":"high-level","description":"next generation","measurement":false},{"type":"zero defect","description":"Phased","measurement":true},{"type":"dedicated","description":"frame","measurement":true},{"type":"4th generation","description":"coherent","measurement":false},{"type":"initiative","description":"Advanced","measurement":false},{"type":"next generation","description":"Multi-tiered","measurement":false},{"type":"Extended","description":"Cross-group","measurement":false},{"type":"standardization","description":"systematic","measurement":false},{"type":"hardware","description":"instruction set","measurement":false}]},
+{"productName":"Ames' Milkvetch","productId":72,"features":["Multi-layered object-oriented budgetary management","Compatible incremental productivity","Sharable actuat
```

Comment:
I believe .disconnect() has be discontinued. You may want to use db.close() instead.